res/lang/bossbetty/bossbetty_en.txt

```diff
@@ -1,153 +1,153 @@
-DIALOGUE_BETTY_LEVEL1_GOODBYE1          = See you again
-DIALOGUE_BETTY_LEVEL1_GOODBYE2          = Hey, I have to go. See you later.
-DIALOGUE_BETTY_LEVEL1_GOODBYE3          = Listen Betty, work is calling. I have to go out for a minute.
+DIALOGUE_BETTY_CHANGETOPIC              = Betty, let's talk about something else.
+DIALOGUE_BETTY_LEVEL0_CHANGETOPIC1      = Let's talk about something else.
+DIALOGUE_BETTY_GOODBYE                  = Hmm... see you around.
+DIALOGUE_BETTY_LEVEL0_GOODBYE1          = OK... goodbye!
+DIALOGUE_BETTY_LEVEL1_GOODBYE1          = See you again!
+DIALOGUE_BETTY_LEVEL1_GOODBYE2          = Hey, I have to go. See you later!
+DIALOGUE_BETTY_LEVEL1_GOODBYE3          = Listen Betty, work's calling. I have to go out for a bit.
 
-BETTY_PRESENT_1            = foot spray
-BETTY_PRESENT_2            = dinner for two
-BETTY_PRESENT_3            = nose surgery
-BETTY_PRESENT_4            = custom written novel
-BETTY_PRESENT_5            = pearl necklace
-BETTY_PRESENT_6            = dwarf hamster coat
-BETTY_PRESENT_7            = necklace with large blood diamond
-BETTY_PRESENT_8            = sports car
-BETTY_PRESENT_9            = ring
-BETTY_PRESENT_10           = yacht
+BETTY_PRESENT_1            = Foot spray
+BETTY_PRESENT_2            = Dinner for two
+BETTY_PRESENT_3            = Nasal plastic surgery
+BETTY_PRESENT_4            = Personalized novel
+BETTY_PRESENT_5            = Pearl necklace
+BETTY_PRESENT_6            = Dwarf hamster coat
+BETTY_PRESENT_7            = Large red diamond necklace
+BETTY_PRESENT_8            = Sports car
+BETTY_PRESENT_9            = Diamond ring
+BETTY_PRESENT_10           = Yacht
 
 
-AWARDNAME_CULTURE            = Sammy "for Culture"
-AWARDNAME_NEWS               = Sammy "for News"
-AWARDNAME_AUDIENCE           = Sammy "for Audience"
-AWARDNAME_CUSTOMPRODUCTION   = Sammy "for custom productions"
+AWARDNAME_CULTURE            = Sammy for Culture
+AWARDNAME_NEWS               = Sammy for News
+AWARDNAME_AUDIENCE           = Sammy for Audience
+AWARDNAME_CUSTOMPRODUCTION   = Sammy for custom productions
 AWARD_WON                    = Award won
 YOU_WON_THE_AWARDNAME        = You won the %AWARDNAME%.
 PLAYERNAME_WON_THE_AWARDNAME = %PLAYERNAME% has won the %AWARDNAME%.
 NOBODY_WON_THE_AWARDNAME     = Due to lack of participants the Sammy has not been awarded.
 CHANNEL_IMAGE                = Channel image
 FOR_X_HOURS                  = for %X% hours
 FOR_X_HOUR                   = for %X% hour
-ATTRACTIVITY                 = attractivity
+ATTRACTIVITY                 = attractiveness
 
 
 DIALOGUE_BOSS_MAIN_TITLE_DEFAULT1                = What's up %PLAYERNAME%?
-DIALOGUE_BOSS_MAIN_TITLE_DEFAULT2                = %PLAYERNAME%, you again ?!
-DIALOGUE_BOSS_MAIN_TITLE_UNFRIENDLY1             = You ... you dare to ask?!
-DIALOGUE_BOSS_MAIN_TITLE_UNFRIENDLY2             = Are you still in your right mind?
+DIALOGUE_BOSS_MAIN_TITLE_DEFAULT2                = %PLAYERNAME%, you again?
```

Comment:
?! on purpose ... the boss is always angry, so each question is also a "once again ... you!"

Did you remove it to keep the "?!" for the even more unfriendly texts?

Reply:
Yes.  On this occasion, it seemed like the boss should be annoyed, but not apoplectic.